ArduCopter/ReleaseNotes.txt

```diff
@@ -1,5 +1,200 @@
+        iv) Range finder range used to protect against premature release (see PLDP_RNG_MIN)
+        v) Touchdown detection threshold is configurable (see PLDP_THRESH)
+    i) Position controller angle max adjusted inflight with CH6 Tuning knob (set TUNE=59)
+    j) Surface tracking time constant allows tuning response (see SURFTRAK_TC)
+    k) Takeoff throttle max is configurable (see TKOFF_TH_MAX)
+    l) Throttle-Gain boost increases attitude control gains when throttle high (see ATC_THR_G_BOOST)
+    m) Waypoint navigation cornering acceleration is configurable (see WPNAV_ACCEL_C)
+    n) WeatherVane option to only use during takoffs and landings (see WVANE_ENABLE)
+6) TradHeli specific enhancements
+    a) Manual autorotation support
+    b) Improved collect to yaw compensation
+7) Camera and gimbal enhancements
+    a) BMMCC support included in Servo driver
+    b) DJI RS2/RS3-Pro gimbal support
+    c) Dual camera support (see CAM2_TYPE)
+    d) Gimbal/Mount2 can be moved to retracted or neutral position
+    e) Gremsy ZIO support
+    f) IMAGE_START_CAPTURE, SET_CAMERA_ZOOM/FOCUS, VIDEO_START/STOP_CAPTURE command support
+    g) Paramters renamed and rescaled
+        i) CAM_TRIGG_TYPE renamed to CAM1_TYPE and options have changed
+        ii) CAM_DURATION renamed to CAM1_DURATION and scaled in seconds
+        iii) CAM_FEEDBACK_PIN/POL renamed to CAM1_FEEBAK_PIN/POL
+        iv) CAM_MIN_INTERVAL renamed to CAM1_INTRVAL_MIN and scaled in seconds
+        v) CAM_TRIGG_DIST renamed to CAMx_TRIGG_DIST and accepts fractional values
+    h) RunCam2 4k support
+    i) ViewPro camera gimbal support
+8) Logging changes
+    a) MCU log msg includes main CPU temp and voltage (was part of POWR message)
+    b) RCOut banner message always included in Logs
+    c) SCR message includes memory usage of all running scripts
+    d) CANS message includes CAN bus tx/rx statistics
+    e) OFCA (optical flow calibration log message) units added
+    f) Home location not logged to CMD message
+    g) MOTB message includes throttle output
+9) Scripting enhancements
+    a) Copter deadreckoning upgraded to applet
+    b) EFI Skypower driver gets improved telem messages and bug fixes
+    c) Generator throttle control example added
+    d) Heap max increased by allowing heap to be split across multiple underlying OS heaps
+    e) Hexsoon LEDs applet
+    f) Linter code checks for AP drivers and applets
+    g) Logging from scripts supports more formats
+    h) Parameters can be removed or reordered
+    i) Parameter description support (scripts must be in AP's applet or driver directory)
+    j) Rangefinder driver support
+    k) Runcam_on_arm applet starts recording when vehicle is armed
+    l) Safety switch, E-Stop and motor interlock support
+    m) Scripts can restart all scripts
+    n) Script_Controller applet supports inflight switching of active scripts
+    o) Skipcheck keywords supports skipping argument range checks
```

Comment:
This is probably more a developer thing.

Reply:
ok, I've zapped that line and moved the linter bit down to Developer specfic items.